### ISSUE TITLE
I can't add new blog.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#720]
```
What's the problem?

I can't add new blog. (click authorize scribefire, but nothing happen)
(Scribefire classic is work fine!!)

What browser are you using?

Firefox 18.0 (ubuntu linux)



What version of ScribeFire are you running?

Scribefire next 4.0
```

-----
Original issue reported on code.google.com by `sonny2...@gmail.com` on 14 Jan 2013 at 11:04

**Comment by GoogleCodeExporter** [comment]
```
same issue (wordpress.com)
```

-----
Original comment by `guido.io...@gmail.com` on 15 Jan 2013 at 8:36

**Comment by GoogleCodeExporter** [comment]
```
Resolved in version 4.3.1
```

-----
Original comment by `scribefi...@gmail.com` on 13 Jun 2014 at 6:57
* Changed state: **Fixed**
